bigchaindb/backend/mongodb/connection.py

```diff
@@ -138,7 +139,7 @@ def _connect(self):
         except (pymongo.errors.ConnectionFailure,
                 pymongo.errors.OperationFailure) as exc:
             logger.info('Exception in _connect(): {}'.format(exc))
-            if "Authentication fail" in str(exc):
```

Comment:
Good point. I have only observed this message with X509 auth; I will test with SCRAM and compare.